sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1674 @@
+  extension. intel/llvm#13512 intel/llvm#13924 intel/llvm#14743
+- Added support for `get_backend_info` API into various SYCL classes (`platform`, `context`, etc.). intel/llvm#12906
+- Implemented [`sycl_ext_oneapi_group_load_store`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/proposed/sycl_ext_oneapi_group_load_store.asciidoc).
+  Please note that the implementation is naive and does not expose any special
+  HW capabilities, it won't provide any performance benefit over how a group
+  load/store could be done without this extension using simple `for` loop and
+  group barriers. intel/llvm#13043
+- Implemented [`sycl_ext_codeplay_enqueue_native_command`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_codeplay_enqueue_native_command.asciidoc) extension. intel/llvm#14136
+- Added initial support for [`sycl_ext_oneapi_free_function_kernels`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/proposed/sycl_ext_oneapi_free_function_kernels.asciidoc) extension. intel/llvm#13207 intel/llvm#13885
+  Known limitations:
+    - free function kernels are only supported if defined at file scope
+    - `SYCL_EXTERNAL` has to be used alongside `SYCL_EXT_ONEAPI_FUNCTION_PROPERTY`
+      to define free function kernel
+    - compiler won't emit any diagnostics if some restrictions from the
+      extension specification are violated
+    - arguments of a free function kernels cannot be composite data types like
+      structs or SYCL classes like `accessor`
+    - using `-fsycl-dead-args-optimization` (ON by default) can lead to failures
+    - `info::kernel::num_args` won't return the right result for free function
+      kernels
+- Added experimental ESIMD function `fma` which results in a guaranteed fused
+  multiply-add operation performed. intel/llvm#13366
+- Implemented revision 2 of
+  [`sycl_ext_oneapi_group_sort`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_group_sort.asciidoc)
+  extension. intel/llvm14399 intel/llvm#14185 intel/llvm#13942 intel/llvm#13908
+  intel/llvm#14591
+
+
+### SYCLcompat library
+
+- Added support for 2-byte and 4-byte `memset` operations. intel/llvm#13409
+- Added `compare`, `compare_both`, `compare_mask` and their `unordered_*`
+  counterparts. intel/llvm#12998
+- Added APIs for performing arithmetic operations on 33-bit extended values. intel/llvm#13006
+- Added APIs for performing bitwise operations on 33-bit extended values. intel/llvm#13727
+- Added `device_count` and `get_device_id` utility APIs. intel/llvm#14013
+- Added experimental `launch` API overloads that accept sub-group size. intel/llvm#13767
```

Comment:
These APIs (#13767) were introduced in a highly experimental state that has already been removed from syclcompat (thought that change didn't make it into this release). Is it reasonable to remove it from the changelog to avoid exposing this functionality? Sorry if it's a dumb question.

Reply:
We control what we want to "advertise" and if something is too unstable/unfinished to be mentioned we can for sure drop it, especially if we know that we decided to drop that. It is just because I'm not familiar with `syclcompat` I listed everything there was

Reply:
I see. I was not entirely aware of what the process was. I'd suggest to remove it from the release notes then. Thanks!

Reply:
```suggestion
```

Reply:
Removed in cf3fb5e9e9be153e83c3e058e053012aa1b3a5f0